### DISCUSSION TITLE
Avoiding "session expired"

### DISCUSSION THREAD

**Discussion by predmijat** [go-shiori/shiori#592] [Q&A]
Hi,

I'm running shiori inside a Docker container and every time I try to save a bookmark (after adding all the tags) I get "session expired" - then I have to go to extension settings, log in, go back and add all the tags again and then it saves it. I'm running Firefox. Is there a way to keep the session running forever?

Thanks!

**Comment by fmartingr** [comment]
Hey @predmijat, sorry to hear. Currently session management is a bit simplistic. I'm trying to evolve it into something better with #497 but that might take a while, though I had some good progress the past weeks. Once that's done sessions will be much more powerful and useable, so a simple update to the extension with the new parameters should be enough to fix this nuisance.
